source/blog/2018-12-14-the-ember-times-issue-77.md

```diff
@@ -1,14 +1,14 @@
 ---
 title: The Ember Times - Issue No. XX
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/2018/12/14-the-ember-times-issue-77.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>, want to learn more about component patterns 📚🧐? 
```

Comment:
qq here just for the purpose of clarity so I don't misunderstand this, I'm assuming `<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>` ideally should be deleted and replaced with the catchy intro